tools/benchmark_tool/openvino/tools/benchmark/benchmark.py

```diff
@@ -60,7 +60,12 @@ def set_cache_dir(self, cache_dir: str):
         self.core.set_property({'CACHE_DIR': cache_dir})
 
     def set_allow_auto_batching(self, flag: bool):
-        self.core.set_property({'ALLOW_AUTO_BATCHING': flag})
+        try:
+            self.core.set_property({'ALLOW_AUTO_BATCHING': flag})
```

Comment:
In C++ this flag was moved to device config. We should align it https://github.com/openvinotoolkit/openvino/blob/71dcdf8a28b0c432e835e5ec8d52bf34fa07a7a6/samples/cpp/benchmark_app/main.cpp#L553

Reply:
Aligned